cosmo_tester/test_suites/test_snapshots/snapshots_multiple_managers_test.py

```diff
@@ -0,0 +1,135 @@
+import random
+import string
+import tempfile
+import os
+
+from cloudify_rest_client.exceptions import CloudifyClientError
+
+from cosmo_tester.framework.cfy_helper import CfyHelper
+from cosmo_tester.framework.util import create_rest_client
+from cosmo_tester.test_suites.test_blueprints.nodecellar_test import (
+    OpenStackNodeCellarTest)
+
+
+class TwoManagersTest(OpenStackNodeCellarTest):
```

Comment:
There is one test in the class OpenStackNodeCellarTest which is sufficient for this purpose. The class TwoManagersTest only overrides some methods.